.github/workflows/coverage.yml

```diff
@@ -0,0 +1,37 @@
+name: Coverage check
+
+on:
+  push:
+    branches:
+      - main 
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch: {}
+
+jobs:
+  coverage:
+    runs-on: ${{ matrix.os }}-latest
+    strategy:
+      matrix:
+        os: [ubuntu]
```

Comment:
Do we not need a version here? https://github.com/actions/runner-images#available-images